packages/eslint-plugin-js/rules/indent/indent.test.ts

```diff
@@ -17,16 +17,25 @@ import rule from './indent'
 const fixture = readFileSync(join(__dirname, './fixtures/indent-invalid-fixture-1.js'), 'utf8')
 const fixedFixture = readFileSync(join(__dirname, './fixtures/indent-valid-fixture-1.js'), 'utf8')
 
+type ErrorInput = [number, number | string, number | string, string]
+interface ErrorOutput {
+  messageId: string
+  data: {
+    expected: string
+    actual: string | number
+  }
+  type: string
+  line: number
+}
+
 /**
  * Create error message object for failure cases with a single 'found' indentation type
- * @param {string} providedIndentType indent type of string or tab
- * @param {Array} providedErrors error info
- * @returns {object} returns the error messages collection
- * @private
```

Comment:
Removing the types from the JSDoc is enough. Keeping comments should be better

Reply:
I don't see them as very helpful, as the variable names are already self-explaining. I don't usually remove them but since this functions has a bit weird overloads, I feel it's better to remove them instead of duplicate those messages.